examples/de-aws-eks/README.md

```diff
@@ -38,13 +38,13 @@ You will need an AWS account, and a [profile set for it](https://docs.aws.amazon
 You can use the AWS CLI to set this up by running `aws configure --profile document-engine-example`. After running this command, the configuration wizard will guide you through setting up the profile. 
 
 Or you can manually edit the `~/.aws/*` configuration files. 
-This entails something like this in `~/.aws/config`:
+This entails something like this in `~/.aws/credentials`:
```

Comment:
Not both?

Reply:
The specific example in this readme is from `~/.aws/credentials`